stockpile/roles/ceph/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+
+- name: see if ceph installed
+  shell: "rpm -q ceph-common"
+  ignore_errors: true
+  register: ceph_present
+
+# the next few tasks run ceph commands 
+# and capture the json output
+
+- name: ceph version
+  shell: "ceph -f json version"
+  register: ceph_version
+  when: ceph_present.rc == sts_ok
```

Comment:
why not just do ceph_present.rc == 0 ? Any particular reason behind this approach?

Reply:
just trying to make it as clear as possible, but ceph_present.rc == 0 works just fine.  Does it make it easier if we don't have defaults/main.yml file?  I couldn't see a way to define a var for this in tasks/main.yml.

Reply:
would be nice if stockpile.yml defined this var, it just makes it clearer what we are checking for.  In general, it's bad form to use hardcoded numbers like 0 in code, where the reader may not know what they mean.

Reply:
Understood @bengland2 , I've seen quite a lot of code with $random_var.rc == 0 that it looked a bit out for me. Also, https://github.com/redhat-performance/stockpile/blob/master/stockpile/group_vars/all.yml is where it'll go if you wanted stockpile to define vars.